src/Kiota.Builder/CodeDOM/CodeMethod.cs

```diff
@@ -303,6 +303,11 @@ public CodeTypeBase GetDiscriminatorMappingValue(string key)
             return value;
         return null;
     }
+    public void RemoveDiscriminatorMapping(params string[] keys) {
+        if(keys == null || !keys.Any()) throw new ArgumentNullException(nameof(keys));
```

Comment:
Maybe this should be only a null check because in the case of empty keys, the function would be no-op.

But if we want to capture it as an error condition, I think we should throw `ArgumentException` instead of `ArgumentNullException`.
```suggestion
        if(keys == null || !keys.Any()) throw new ArgumentException("keys cannot be null or empty", nameof(keys));
```

Reply:
agreed, empty array is no-op, so no need to test for it. As for the null scenario, I've used the static method instead which results in fewer branches and better code coverage (without having to implement a unit test specifically for it)